src/Popper.jsx

```diff
@@ -139,13 +139,15 @@ class Popper extends Component {
       left,
       position,
     } = this.state.data.offsets.popper
+    const styles = this.state.data.styles
 
     return {
       position,
       top:        0,
       left:       0,
       transform:  `translate3d(${left}px, ${top}px, 0px)`,
       willChange: 'transform',
+      ...styles,
```

Comment:
FWIW, this is similar to how PopperJS handles things inside of its own `applyStyle` modifier.

https://github.com/FezVrasta/popper.js/blob/master/src/popper/modifiers/applyStyle.js#L53